admin/class-admin.php

```
@@ -432,8 +434,24 @@ public function enqueue_scripts() {
 					'saveFailed'               => __( 'Failed to save settings.', 'faz-cookie-manager' ),
 					'loadFailed'               => __( 'Failed to load settings.', 'faz-cookie-manager' ),
 					'confirmDelete'            => __( 'Are you sure you want to delete this?', 'faz-cookie-manager' ),
+					// Banner page — banner-status toggle (loadBannerEnabledToggle).
+					'banner'                   => array(
```

PHPStan (2.1.54)

[error] Array has 2 duplicate keys with value 'banner' ('banner', 'banner').


(array.duplicateKey)